README.md

```diff
@@ -21,8 +21,7 @@ before you continue with the build process. This project is tested
 with Clang, and GCC occasionally. For example:
 
 ```
-export CC=clang
-export CXX=clang++
+cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/clang.cmake ..
```

Comment:
It should be noted that if fmtlib or GTest are not compiled with libc++, then this might vomit lots of linker errors.

Reply:
Indeed.
I guess the toolchain should be global for the whole workspace.
Eventually, for the warnings it could be specified for the project, but all the libc++ stuff should be global to the other projects.